docs/markdown/Users.md

```diff
@@ -4,10 +4,11 @@ title: Users
 
 # List of projects using Meson
```

Comment:
Should this be changed to "Notable projects using Meson"? (maybe at the same time the list is trimmed).

Reply:
Done (but no trimming -- I don't want to do that right now).